.circleci/build_dependencies.sh

```diff
@@ -34,6 +34,7 @@ sudo zypper ar -f https://download.opensuse.org/repositories/devel:/openQA:/Leap
 tools/retry sudo zypper --gpg-auto-import-keys ref
 sudo zypper -n install openQA-devel
 sudo zypper -n install perl-TAP-Harness-JUnit
+sudo zypper -n install docker
```

Comment:
I would prefer if we can avoid this. We should not duplicate test dependencies manually in a shell script. We have either `.circleci/ci-packages.txt` or `dependencies.yaml`. I don't know why `perl-TAP-Harness-JUnit` is specified manually here but I doubt we need docker mentioned here.

Reply:
When I remember correctly, `perl-TAP-Harness-JUnit` was only used within the CI at some point. For Docker counts the same as we would likely not want to require every contributor to install it. We'd needed to add another category to `dependencies.yaml` and update the update script to cover this file. I don't think it is worth it.

Reply:
I've removed this line